sklearn/preprocessing/_discretization.py

```diff
@@ -188,6 +188,11 @@ def fit(self, X, y=None):
         self.bin_edges_ = bin_edges
         self.n_bins_ = n_bins
 
+        if 'onehot' in self.encode:
+            self._ohe_encoder = OneHotEncoder(
```

Comment:
let's just call it _encoder, so that we might use the same `inverse_transform` code when unary encoding is available.